ribs/emitters/_evolution_strategy_emitter.py

```diff
@@ -0,0 +1,187 @@
+"""Provides the EvolutionStrategyEmitter."""
+import itertools
+
+import numpy as np
+
+from ribs.emitters._emitter_base import EmitterBase
+from ribs.emitters.rankers import RankerBase
+
+
+def get_ranker(ranker_name):
+    try:
+        ranker = globals()[ranker_name]()
```

Comment:
I think this is a bit unpredictable as there are global variables like `np` that may be detected here.